sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj

```diff
@@ -42,4 +42,9 @@
     <Compile Include="$(AzureCoreSharedSources)PemReader.cs" LinkBase="Shared" />
   </ItemGroup>
 
+  <!--Temporary until next Core release-->
```

Comment:
Nit: link to tracking issue in comment?

Reply:
Will be reverting this.

Reply:
If you wouldn't, please also remove the ProjectRef from the sln. I can always do it if you forget, though. Not a big issue. Just means I can run all tests faster. :)